.github/CODEOWNERS

```diff
@@ -5,6 +5,8 @@
 *.gn @youtube/cobalt-build
 *.gni @youtube/cobalt-build
 
+/media/ @youtube/cobalt-media
```

Comment:
I guess I'm unsure if you want this
on line 3 or line 8. The last match supersedes.